basic_bootloader/src/bootloader/transaction/mod.rs

```diff
@@ -892,6 +995,136 @@ impl<T: 'static + Clone + Copy + core::fmt::Debug> ParsedValue<T> {
     }
 }
 
+pub struct AccessListParser {
```

Comment:
To think about: maybe we should move it to separate file and add some tests?

Reply:
Maybe move to a different file makes sense. I added a test in the rig/utils, as I needed a way to encode. I can add some vectors tho

Reply:
Moved to different file, added some test vectors